arch/arm/src/common/arm_assert.c

```diff
@@ -520,39 +489,12 @@ void up_assert(const char *filename, int lineno)
 {
   board_autoled_on(LED_ASSERTION);
 
-  /* Flush any buffered SYSLOG data (prior to the assertion) */
-
-  syslog_flush();
-
-  _alert("Assertion failed "
-#ifdef CONFIG_SMP
-         "CPU%d "
-#endif
-         "at file:%s line: %d"
-#if CONFIG_TASK_NAME_SIZE > 0
-         " task: %s"
-#endif
-         "\n",
-#ifdef CONFIG_SMP
-         up_cpu_index(),
-#endif
-         filename, lineno
-#if CONFIG_TASK_NAME_SIZE > 0
-         , running_task()->name
-#endif
-        );
-
 #ifdef CONFIG_ARCH_STACKDUMP
   arm_dumpstate();
```

Comment:
can we share all arm_dumpstate code except arm_registerdump?

Reply:
Maybe another patch?  this is also lots of modification

Reply:
Another patch, it brings more modification in other source files